lib/fhir_models/bootstrap/json.rb

```diff
@@ -24,5 +24,13 @@ def self.from_json(json)
       end
       resource
     end
+
+    def self.module_version_name
+      FHIR.module_version_name
+    end
+
+    def self.module_version
+      FHIR.module_version
```

Comment:
What I was trying to accomplish here was deferring the version to the FHIR module whenever methods are called through FHIR instead of through a model.  i.e., when making a new resource from a json/xml, we can call `FHIR::Account.new(*payload*)` and the version will be inferred from the Account class.  But if you run FHIR::Json.from_json (as we do, at least in our unit tests), it has no class to find the version from.  The FHIR module defaults to R4 and these just retrieve that.  

That being said, I'm sure there is a cleaner way to model this behavior.  If there is a cleaner inheritance structure that is preferred, I can make that change.

Reply:
I believe I've made the changes to allow what you are proposing, but the inheritance looks a little funky.  I would appreciate a closer look at r4/fhir_ext/json.rb to make sure that seems alright, especially the set up of FHIR::Json to default to r4.